docs/changelog.md

```diff
@@ -40,6 +40,10 @@ permalink: /changelog/
       * Labels in labeled metrics now have a correct, and slightly stricter, regular expression.
         See [label format](https://mozilla.github.io/glean/user/metrics/index.html#label-format) for more information.
 
+* **feature-app-links**
+  * ⚠️ **This is a breaking change**: `alwaysAllowedSchemes` is removed as a parameter for `AppLinksInterceptor`.
+  * Added `engineSupportedSchemes` as a parameter for `AppLinksInterceptor`.  This allows caller to specifies which protocol is supported by the engine.
+    * Using this information, app links can decide if a protocol should be launched in third party or not regardless of user preference.
```

Comment:
nit:  if a protocol should be launched in **a** third party **app** or not regardless of user preference.

Reply:
Thanks